pkg/graph/schema.graphql

```diff
@@ -1575,7 +1574,6 @@ enum TRBCollabGroupOption {
   CLOUD
   PRIVACY_ADVISOR
   GOVERNANCE_REVIEW_BOARD
-  OTHER
```

Comment:
Is there a reason other was removed? I think the front end will still need it to conditionally render the `collabGroupOther` field